toes/extra_toe.py

```diff
@@ -0,0 +1,22 @@
+import random
+from discord import app_commands as slash, Client, Interaction, TextChannel
+from util.debug import DEBUG_GUILD
+
+DEBUG = False
+
+# This command will have the bot send a message to a specified text channel 
+# bot will announce the user who initiated the command in the current channel
+@slash.command()
+async def say(interaction: Interaction, channel: TextChannel, message:str,) -> None:
+    '''Have Kyoyobot say something in a channel.'''
+    
+    text = f'{interaction.user} told me to say \"{message}\" in #{channel.name}\n'
```

Comment:
Maybe we should make these mentions? i.e. ping the user with <@id> and the channel with <#id>

Reply:
Good idea, I added the mentions.